src/Api/Droplet.php

```diff
@@ -27,14 +27,24 @@ class Droplet extends AbstractApi
      */
     public function getAll()
     {
-        $droplets = $this->adapter->get(sprintf('%s/droplets?per_page=%d', self::ENDPOINT, PHP_INT_MAX));
-        $droplets = json_decode($droplets);
-
-        $this->extractMeta($droplets);
-
-        return array_map(function ($droplet) {
+        $thereAreNewResults = true;
+        $results = [];
+        $url = sprintf('%s/droplets?per_page=%d', self::ENDPOINT, PHP_INT_MAX);
```

Comment:
@funkatron I tried. They ignore any value > 200. As you can see, it is like this in the original module and it's not make any difference.